app/scripts/services/rest.js

```diff
@@ -2,8 +2,9 @@ export function Api ($rootScope, $resource, config) {
   // fetch API server details
   let protocol = config.protocol
   let host = config.host
+  let hostPath = config.hostPath.startsWith('/') ? config.hostPath.replace(/\/$/, '') : '/' + config.hostPath.replace(/\/$/, '')
```

Comment:
Can we just use a single regex for this?

Reply:
@andrew-w-ross not sure how you would do this using regex, but there is a nifty string library which can do this check and replace, so have included and used it instead.